src/components/atoms/UserStatusDropdown/UserStatusDropdown.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import { Experience, Statuses } from "types/User";
+
+import "./UserStatusDropdown.scss";
+
+export interface UserStatusDropdownProps {
+  options: Array<Statuses>;
+  onChange: (option: string) => void;
+  label: string | Experience;
+}
+
+export const UserStatusDropdown: React.FC<UserStatusDropdownProps> = ({
+  options,
+  label,
+  onChange,
+}) => {
+  const [isOpen, setIsOpen] = useState(false);
```

Comment:
Can we reuse `useShowHide` hook? Based on the logic, we can.

Reply:
Plus make sure to 'rename' the exported data from `useShowHide` so it has 'contextual/semantically correct names' for its purpose.